src/pages/version/Banners/ErrorBanner/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+import Callout from "@leafygreen-ui/callout";
+import { DisplayModal } from "components/DisplayModal";
+import Icon from "components/Icon";
+import {
+  BannerContainer,
+  TitleWrapper,
+  OrderedList,
+  ListItem,
+  ModalTrigger,
+} from "../styles";
+
+interface ErrorBannerProps {
+  errors: string[];
+}
+
+const ErrorBanner: React.VFC<ErrorBannerProps> = ({ errors }) => {
+  const [showModal, setShowModal] = useState(false);
+
+  const errorTitle =
+    errors.length === 1
+      ? "1 error in configuration file"
+      : `${errors.length} errors in configuration file`;
+
+  return (
+    <BannerContainer data-cy="configuration-errors-banner">
+      <Callout variant="warning" title={errorTitle.toUpperCase()}>
+        {errors[0]}
```

Comment:
I'm not sure I'm convinced that we should show a full error here. Since the error banner isn't dismissible, it results in the banner taking up a large portion of the screen, and could persist for users who aren't really able to take action on it. These banners are already a lot more prominent on the page than the legacy implementation, so I'm hesitant to claim even more screen real estate. But what do you think?

Reply:
My only hang-up is how the Callout looks oddly bare (like something's missing) if we take away the full error:

<img width="1492" alt="Screenshot 2023-03-01 at 6 39 13 PM" src="https://user-images.githubusercontent.com/47064971/222455938-560ed9fa-cdbc-47a8-af06-a739c49ec304.png">

I'll ask more about this in design sync today :>